### PR TITLE
JitArm64: Call rlwinmx implementation from rlwnmx with imm b

### PR DESCRIPTION
**Pull request by JosJuice**
This way we get some additional optimized cases for rlwnmx with imm b.

### CODE REVIEW

File: Source/Core/Core/PowerPC/JitArm64/Jit.h [hunk 1]
```diff
@@ -182,6 +182,8 @@ class JitArm64 : public JitBase, public Arm64Gen::ARM64CodeBlock, public CommonA
 
   bool IsFPRStoreSafe(size_t guest_reg) const;
 
+  void rlwinmx_internal(UGeckoInstruction inst, u32 sh);
```
Comment (lioncash) [inline Source/Core/Core/PowerPC/JitArm64/Jit.h:185]: Is this intended to be public?

Comment (JosJuice) [reply Source/Core/Core/PowerPC/JitArm64/Jit.h:185]: Well, I guess it has no real reason to be public... All these other helper methods already were, though. What do you think, should I move this one to private?

Comment (lioncash) [reply Source/Core/Core/PowerPC/JitArm64/Jit.h:185]: tbh, the other respective helpers should also reasonably be private if they have no need to be public. I'm fine merging as is, if you'd prefer to do that as it's own thing

Comment (JosJuice) [reply Source/Core/Core/PowerPC/JitArm64/Jit.h:185]: Yeah, I think I'll do that in a separate PR then.

**Review by lioncash — APPROVED** [review]

**Pull request merged**